src/options.toml

```diff
@@ -126,11 +165,11 @@
         #If one of these options is not used the program will exit early.
         solver = "GMRES"
 [Mesh]
-    #Serial refinement level
+    #Serial uniform refinement level
     ref_ser = 0
-    #Parallel refinement level
+    #Parallel uniform refinement level
     ref_par = 0
-    #The polynomial refinement/order of our shape functions
+    #The polynomial order of our shape functions
     prefinement = 1
```

Comment:
This might be easier to read if it were `p_refinement` rather than `prefinement`

Reply:
Yeah, I'd agree with that being a good change.